ChangeLog

```diff
@@ -2,6 +2,12 @@ mbed TLS ChangeLog (Sorted per branch, date)
 
 = mbed TLS x.x.x branch released xxxx-xx-xx
 
+Features
+   * Add support for fragmentation of outgoing DTLS handshake messages. This
+     is controlled by the maximum fragment length as set locally or negotiated
+     with the peer, as well as new per-connection MTU option, set using
```

Comment:
Minor: Is there a `by` missing in the last part? `... with the peer, and by the new per-connection MTU option, ...`?

Reply:
I fee like it could work both ways, but I think it's indeed clearer with a "by" in the second part. Adding it.